pages/addCommunity.tsx

```diff
@@ -142,10 +149,11 @@ const AddCommunity = (): JSX.Element => {
             <Label>
               Members:
               <Input
-                type="number"
+                type="text"
```

Comment:
Why we changed this?

Reply:
maxLength doesn't work with type number!